ndx-whisk/ndx-meta.yaml

```
@@ -0,0 +1,8 @@
+name: ndx-whisk
+version: 0.1.0
```

```suggestion
version: 0.1.1
```
The latest version appears to be 0.1.1